components/analytics/system_analytics/system_analytics.tsx

```diff
@@ -43,8 +45,15 @@ export default class SystemAnalytics extends React.PureComponent {
         }
     }
 
-    render() {
-        const stats = this.props.stats;
+    private getStatValue(stat: number | AnalyticsRow[]):number {
+        if (typeof stat === 'number') {
+            return stat;
+        }
+        return stat[0].value;
+    }
```

Comment:
The problem is not with the enterprise code, but that we are assuming there is a first element always and it might be undefined.

```suggestion
    private getStatValue(stat: number | AnalyticsRow[]):number? {
        if (typeof stat === 'number') {
            return stat;
        }
        if (!stat) {
            return undefined;
        }
        return stat[0].value;
    }
```
by using undefined, you'll have to deal with it when presenting the data as being "not available".

This is one possible approach, but there are other options, so don't feel forced to use this one if you come up with some other idea.